src/components/buttons/_buttons.scss

```diff
@@ -0,0 +1,165 @@
+@mixin buttonHover($color1, $color2, $percent) {
+  &:hover:not([disabled]),
+  &:focus:not([disabled]),
+  &:active:not([disabled]),
+  &:active:focus:not([disabled]) {
+    @include sgButtonHoverMix($color1, $color2, $percent);
+  }
+}
+
+.sg-button {
+  position: relative;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  background-color: $black;
+  height: 40px;
+  border-radius: 20px;
+  border: none;
+  white-space: nowrap;
+  text-decoration: none;
+  font-size: 15px;
+  line-height: 15px;
+  font-weight: bold;
+  color: $white;
+  text-transform: uppercase;
+  padding: 0 spacing(m);
+  will-change: background-color, border-color;
+  transition-property: background-color, border-color;
+  transition-duration: 0.2s;
+  transition-timing-function: ease-in-out;
+  cursor: pointer;
+  box-sizing: border-box;
+
+  // Added to align text in the center,
+  // as soon as Proxima is fixed we could remove it.
+  &__text {
+    position: relative;
+    top: 1px;
+  }
+
+  &__icon {
+    fill: $white;
+    display: inline-flex;
+    margin-right: spacing(xs);
+
+    &--small {
+      margin-right: spacing(xs);
+    }
+
+    &--large {
+      margin-right: spacing(s);
+    }
+  }
+
+  &:focus {
```

Comment:
for a11y I would recommend not to add outline: none, here is why: https://developers.google.com/web/fundamentals/accessibility/how-to-review?utm_source=lighthouse&utm_medium=extension -> "...It’s common practice to disable focus styles without providing an alternative by using outline: none in CSS, but this is an anti-pattern. If a keyboard user can’t see what’s focused, they have no way of interacting with the page." wdyt @burakukula ? :)

Reply:
@ElaLukaszewicz sure thing! good catch 👍